index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="main.css" />
+    <title>Document</title>
```

Comment:
- [ ] It is a good idea to provide a descriptive title for your page because it is important for usability or social sharing. The title tag of a web page is meant to be an accurate and concise description of a page's content.